### PR TITLE
Tutorial 8 revamp

### PR DESCRIPTION
**Pull request by jrgissing**
Tutorial 8 updates:
- [x] Single-periodic CNT
- [x] Polystrene matrix
- [x] Update to smaller system
- [x] Update manuscript for smaller system

### CODE REVIEW

**Comment by jrgissing** [comment]
@simongravelle this is ready for you to take a glance at it. I put 1000 styrene molecules in the box. With the CNT, the system is 17,000 atoms. Please let me know if that is within your upper limit for system size.

**Comment by jrgissing** [comment]
also, I did not update the plots

**Comment by simongravelle** [comment]
> @simongravelle this is ready for you to take a glance at it. I put 1000 styrene molecules in the box. With the CNT, the system is 17,000 atoms. Please let me know if that is within your upper limit for system size.

Great, I will take a look. Simon

**Comment by simongravelle** [comment]
@jrgissing I love the dump image generated with the CNT in the center, it looks nice. 

The simulation is indeed a bit heavy, the heaviest of the tutorials by far (I note that the runs are relatively short, which overall makes it kind of OK, despite the number of atoms). Just to be sure: Is there a fundamental reason for the initial system to be that large? If not, and if by chance it's possible to reduce a bit the number of atoms, that would make the tutorial more accessible to interested people....

**Comment by jrgissing** [comment]
Note that I was running on 32 procs, I think. I assume the other runs were in serial? The length in the CNT axial direction was just because that was the length of the CNT in the LAMMPS example. So, it would be relatively easy to reduce that, maybe by a factor of 4 or 5? The other two dimensions probably should not be reduced if you want a similar dump image. I originally was hoping to get away with this large system just to have decent lengths/statistics for the polymer chains.

**Comment by simongravelle** [comment]
A length reduction is definitely something to consider. If a factor of 5 really kills the statistics, then may be could reduce it simply by a factor of 2 or 3 as a compromise?

I feel that these tutorials will be more useful if they can be launched by anyone with a normal laptop.

**Comment by jrgissing** [comment]
@simongravelle I think this is ready for you to take a look at. the systems now run in 3-4 minutes on 1 processor. Can you regenerate the density evolution and reaction count plots?

**Comment by simongravelle** [comment]
> @simongravelle I think this is ready for you to take a look at. the systems now run in 3-4 minutes on 1 processor. Can you regenerate the density evolution and reaction count plots?

Awesome, yes, let me do that now.

File: lammps-tutorials.tex [hunk 1]
```diff
@@ -4212,16 +4225,14 @@ \subsubsection{Simulating the reaction}
...
-S.G. conceived and wrote all the online tutorials and underlying Sphinx documentation
-for \href{https://lammpstutorials.github.io}{lammpstutorials.github.io}.
-
-\noindent A.K. wrote the LAMMPS GUI software and helped revise the
-tutorials for use with it.
 
-\noindent J.G. is the principal author of fix bond/react and type label
-support in LAMMPS.  He revised tutorials to use type labels.
-% and contributed the eighth tutorial using fix bond/react (?).
+S.G. conceived and wrote the original online tutorials and underlying Sphinx documentation
```
Comment (simongravelle) [inline lammps-tutorials.tex:4229]: @jrgissing @akohlmey I revised the author contribution part to add the tutorial 8 to J.G.'s contrib.  If you don't find the decription fair, please feel free to update.

File: lammps-tutorials.tex [hunk 2]
```diff
@@ -4230,10 +4241,9 @@ \section*{Potentially Conflicting Interests}
 \section*{Funding Information}
```
Comment (simongravelle) [inline lammps-tutorials.tex:4241]: @jrgissing If you have a funding that you feel is relevant, don't hesitate to add it.

**Comment by simongravelle** [comment]
LGTM @jrgissing, I would like to merge if you agree with my changes.

**Pull request merged**